app/services/products_renderer.rb

```diff
@@ -68,8 +68,8 @@ def distributed_products
   end
 
   def products_order
-    if (distributor.preferred_shopfront_product_sorting_method == "by_producer") &&
-       distributor.preferred_shopfront_producer_order.present?
+    if distributor.preferred_shopfront_product_sorting_method ==
+       "by_producer" && distributor.preferred_shopfront_producer_order.present?
```

Comment:
`±` This format is harder to read. 
And I just noticed that it is unnecessary. This was wrapped at 80 chars, but our rubocop_styleguide is set 100.